samples/README.md

```diff
@@ -33,3 +33,25 @@ Useful parameter values:
 ## Components source
 
 All samples use pre-built components. The command to run for each container is built into the pipeline file.
+
+## (Optional) Add sample test coverage
+
+For those samples that cover critical functions of KFP, possibly it should be picked up by KFP's sample test
+so that it won't be broken by accidental PR merge. To do that, the sample to be covered should meet the following 
+requirements/conventions.
+
+* Under the core sample directory `kubeflow/pipelines/samples/core`
+* Make sure that the sample is either `*.py` or `*.ipynb`, and its file name is in consistence with its dir name.
+* For `*.py` sample, its main invokes `kfp.compiler.Compiler().compile()` to compile the pipeline function into pipeline
```

Comment:
This is the recommendation for samples, not required for sample test, isn't it?

Reply:
Wait. I think currently sample test needs to submit a pipeline run. Did I miss something?

Reply:
But this is not a hard requirement for the samples. The sample test infra contains the compilation step.

Reply:
Yes got it.